desktop/src/main/java/bisq/desktop/main/offer/offerbook/OfferBookView.java

```diff
@@ -1289,6 +1281,32 @@ public void updateItem(final OfferBookListItem newItem, boolean empty) {
         return column;
     }
 
+    private PeerInfoIconTrading findOrCreateAvatar(Integer tableRowId, Offer offer) {
```

Comment:
Did you intend to find the avatar and create it only if not found in the map, as you create it all the time right now?

Reply:
@ripcurlx no - I would use a name like `createAvatar`, but I kept this method name to keep it similar with the one in DisputeView.
Do you want me to change it (and maybe the one in DisputeView too)?

Reply:
Yes, please change the other one as well, as it looks like as if some code parts are missing based on the method name.

Reply:
OK
I will commit tomorrow because I want to test something

Reply:
I improved it a bit:
Changed the key that is stored in the map to only contain the offer's ID. Row number + maker's address are not sufficient because the offers keep changing rows/positions (it still worked fine previously but now the map isn't increasing unnecessarily).
The map is cleared on deactivate() to avoid holding entries that may not exist anymore (needed for OfferBookView which keeps changing frequently, not so much for DisputeView).